docs/roadmap.md

```diff
@@ -1,20 +1,76 @@
+---
+title: Roadmap
+description: Public roadmap for Powertools for AWS Lambda (.NET)
+---
+
+<!-- markdownlint-disable MD043 -->
+
 ## Overview
 
-This is our public roadmap that outlines the high level direction we are working towards, namely [Themes](#themes). We update this document when our priorities change: security and stability is our top priority.
+Our public roadmap outlines the high level direction we are working towards. We update this document when our priorities change: security and stability are our top priority.
+
+!!! info "For most up-to-date information, see our [board of activities](https://github.com/orgs/aws-powertools/projects/6/views/14?query=is%3Aopen+sort%3Aupdated-desc){target="_blank"}."
+
+### Key areas
+
+Security and operational excellence take precedence above all else. This means bug fixing, stability, customer's support, and internal compliance may delay one or more key areas below.
+**Missing something or want us to prioritize an existing area?**
+You can help us prioritize by [upvoting existing feature requests](https://github.com/aws-powertools/powertools-lambda-dotnet/issues?q=is%3Aissue%20state%3Aopen%20label%3Afeature-request){target="_blank"}, leaving a comment on what use cases it could unblock for you, and by joining our discussions on Discord.
+
+[![Join our Discord](https://dcbadge.vercel.app/api/server/B8zZKbbyET)](https://discord.gg/B8zZKbbyET){target="_blank"}
+
+### Core Utilities (P0)
+
+#### Logging V2
+
+Modernizing our logging capabilities to align with .NET practices and improve developer experience.
+
+- [ ] Logger buffer implementation
+- [ ] New .NET-friendly API design (Serilog-like patterns)
+- [ ] Correlation ID support via decorator/middleware
+- [ ] JMESPath expression support
+- [ ] RFC: Documentation for SDK context.Logger vs Powertools Logger differences
+
+#### Metrics V2
+
+Updating metrics implementation to support latest EMF specifications and improve performance.
+
+- [ ] Update to latest EMF specifications
+- [ ] Breaking changes implementation
+- [ ] Performance optimizations
+- [ ] API updates
```

Comment:
this reads very generic, which API and what should be updated?

Reply:
API updates - missing functionality that is present in Python implementation (ie: flush_metrics)